src/main.cpp

```diff
@@ -1200,7 +1200,7 @@ bool IsInitialBlockDownload()
     if (lockIBDState)
         return false;
     bool state = (chainActive.Height() < pindexBestHeader->nHeight - 24 * 6 ||
-            pindexBestHeader->GetBlockTime() < GetTime() - 24 * 60 * 60);
+            pindexBestHeader->GetBlockTime() < GetTime() - Params().MaxTipAge());
```

Comment:
Why change this? It only determines whether the client is in initial block download, it shouldn't prevent the client from accepting blocks.